platform-demo-module/src/main/java/com/softicar/platform/demo/module/DemoI18n.java

```diff
@@ -37,6 +41,8 @@
 		.de("Gesperrte Positionen.");
 	I18n0 NET_AMOUNT = new I18n0("Net Amount")//
 		.de("Nettobetrag");
+	I18n0 POINT_IN_TIME = new I18n0("Point-In-Time")//
```

Comment:
I'm somewhat confused now... what's the difference between a _"moment"_ and a _"point in time"_? And an _"instant"_, for that matter? (as in `java.time.Instant`)

Reply:
what do I know? ;) I just needed some names, please suggest better names

Reply:
Okay... it's just the demo module after all. So, never mind.